packages/app/src/resource/BotEditor.tsx

```diff
@@ -64,11 +64,11 @@ export function BotEditor(): JSX.Element | null {
 
   const getCode = useCallback(() => {
     return sendCommand(codeFrameRef.current as HTMLIFrameElement, { command: 'getValue' });
-  }, [codeFrameRef]);
+  }, []);
```

Comment:
This is a good change, but I'm curious how it came up in this PR 🙃

Reply:
When I was removing the restrictions to `ContentTypeString` that I added in this PR before, I happened to catch it